package.json

```diff
@@ -39,6 +39,7 @@
   "dependencies": {
     "amd-name-resolver": "^1.2.0",
     "babel-plugin-transform-es2015-modules-amd": "^6.24.0",
+    "bower": "^1.8.4",
```

Comment:
By default, we don't include bower. It is something that is on-demand installed when required. The reason was, `bower` is quite a large dependency.

Reply:
Not sure how that ended up there - it probably got installed and I just assumed it was okay. Will remove it.